resources/linux/debian/postinst.template

```diff
@@ -81,6 +80,8 @@ if [ "@@NAME@@" != "code-oss" ]; then
 			db_get @@NAME@@/add-microsoft-repo
 			if [ "$RET" = false ]; then
 				WRITE_SOURCE='no'
+			else
```

Comment:
Line 50 of this file should be an AND statement that also checks whether the GPG key is in the new location.